NineChronicles.Headless/BlockChainService.cs

```diff
@@ -140,6 +152,26 @@ public UnaryResult<byte[]> GetState(byte[] addressBytes, byte[] blockHashBytes)
             return result.ToDictionary(kv => kv.Key, kv => kv.Value);
         }
 
+        public async UnaryResult<Dictionary<byte[], byte[]>> GetAvatarStatesBySrh(
```

Comment:
I think this method is conflicted with `GetStateBulkBySrh` method.

Reply:
Ah.. It seems like a version with stateroothash of `GetAvatarStates` method that already exists.